docs/introduction.rst

```diff
@@ -4,8 +4,19 @@
 Introduction
 ============
 
-This is the documentation for tskit, the tree sequence toolkit.
+This is the documentation for tskit, the tree sequence toolkit. Tree sequences
+are a highly efficient way of storing a set of related DNA sequences by storing
```

Comment:
Little clunky using 'storing' twice in this sentence. Perhaps use 'encoding' for one of these?